packages/fiori/src/UploadCollection.hbs

```diff
@@ -3,15 +3,17 @@
 	role="region"
 	aria-roledescription="{{_roleDescription}}"
 	@drop="{{_ondrop}}">
-	<div class="ui5-uc-header">
-		<slot name="header"></slot>
-	</div>
```

Comment:
Since the header is deleted the root no longer needs `display: flex;`
Also `<div class="{{classes.content}}">` can be removed, but this will require more work.